articles/link-accounts/user-initiated-linking.md

```diff
@@ -24,33 +24,32 @@ The following is a sample login using Lock:
 ```html
 <script src="${lock_url}"></script>
 <script type="text/javascript">
+
   //Log in using Lock in Redirect Mode
   function login(){
     lock.show();
   }
 
-  //handle redirection from iDP after login
-  $(document).ready(function() {
-    var hash = lock.parseHash();
-    if (hash) {
-      if (hash.error) {
-        alert('There was an error logging in ' + hash.error );
-      } else {
-        ...
-        // the hash comes from the site's first time login
-        lock.getProfile(hash.id_token, function(err, profile) {
-          if (err) {
-            alert('There was an error logging in. ' + err);
-          } else {
-            // Save the JWT token.
-            localStorage.setItem('id_token', hash.id_token);
-            localStorage.setItem('user_id', profile.user_id);
-            showLoggedInUser(profile);
-          }
-        });
+  function lockAuthenticated(authResult)
+  {
```

Comment:
braces should be right after the method `function lockAuthenticated(authResult) {`